CMakeLists.txt

```diff
@@ -186,7 +186,7 @@ if(CLANG_TIDY)
             endif()
             string(REPLACE "/" "_" TARGETNAME "aktualizr_clang_tidy-${SUBDIR}-${FILE}")
             add_custom_target(${TARGETNAME}
-                COMMAND ${CLANG_TIDY} -quiet -header-filter=\(${CMAKE_SOURCE_DIR}|\\.\\.\)/src/.* --extra-arg-before=-Wno-unknown-warning-option -format-style=file -warnings-as-errors=* -p ${CMAKE_BINARY_DIR} ${FILE}
+                COMMAND ${CLANG_TIDY} -quiet -header-filter=\${CMAKE_SOURCE_DIR}/src/.* --extra-arg-before=-Wno-unknown-warning-option -format-style=file -warnings-as-errors=* -p ${CMAKE_BINARY_DIR} ${FILE}
```

Comment:
This probably broke the checks run on our header files. The regex looks horrible but was made to work on Travis and local dev environments: https://github.com/advancedtelematic/aktualizr/pull/680